display_information/add-graphics-with-renderer/README.metadata.json

```
@@ -1,20 +1,30 @@
 {
     "category": "Display information",
-    "description": "A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only affect graphics that do not specify their own symbol style.",
+    "description": "A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only effect graphics that do not specify their own symbol style.",
```

```suggestion
    "description": "A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only affect graphics that do not specify their own symbol style.",
```

I think affect is correct? The renderer would apply an effect which would cause an affect.